UPGRADE_NOTES.md

```diff
@@ -119,6 +119,8 @@ UserForm.create!(name: "Jane")
 
 More info at https://luckyframework.org/guides/saving-with-forms/#passing-data-without-route-params
 
+* Change calls to `form.save_succeeded?` to `form.saved?`
```

Comment:
I'd suggest saying `from` first here. So its:

> Change calls **from** `form.save_succeeded?` to `form.saved?`

Reply:
I agree. I think that reads a bit better

Reply:
✅